src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -15,6 +15,7 @@
 
 from knack.log import get_logger
 from knack.util import CLIError
+from msrest.authentication import BasicTokenAuthentication
```

Comment:
Move it in scope for performance?

Reply:
It is used as a base class for the IMDS authentication class which should be moved into the msrestazure soon, hence I put it at the global scope. But since you have raised it, which is a valid concern, let me move the whole IMDS code to a new file which should eliminate this global import